### PR TITLE
Update config.yml

### PR DESCRIPTION
**Pull request by Jens-Co**
This had me confused, its listed in the wiki tho.

### CODE REVIEW

**Comment by Camotoy** [comment]
Can you also mention that's only for plugins and not applicable to standalone?

**Comment by Jens-Co** [comment]
> Can you also mention that's only for plugins and not applicable to standalone?

hope its fine now :)

File: connector/src/main/resources/config.yml [hunk 1]
```diff
@@ -38,6 +38,8 @@ remote:
   # The IP address of the remote (Java Edition) server
   # If it is "auto", for standalone version the remote address will be set to 127.0.0.1,
   # for plugin versions, Geyser will attempt to find the best address to connect to.
+  # By keeping it as auto, the address, port, and Floodgate support will be automatically configured.
+  # In standalone, keeping this as auto sets the remote address to 127.0.0.1 therefore Floodgate will not be Automatically configurated.
```
Comment (Chew) [inline connector/src/main/resources/config.yml:42]:
```suggestion
  # In standalone, keeping this as auto sets the remote address to 127.0.0.1, and Floodgate will not be automatically configured.
```

Comment (Chew) [reply connector/src/main/resources/config.yml:42]: I still feel like it's the wrong conjunction word. Either way, it needs a comma.

**Comment by Camotoy** [comment]
You can probably take the two sentences and combine them: `By keeping it as auto, the address, port, and Floodgate support will be automatically configured on plugin versions.` We already address standalone above.

**Comment by Jens-Co** [comment]
> You can probably take the two sentences and combine them: `By keeping it as auto, the address, port, and Floodgate support will be automatically configured on plugin versions.` We already address standalone above.

Made changes :)

**Comment by Camotoy** [comment]
Addressed in 9c607af instead. Thank you so much anyway!

**Pull request closed without merging**